src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -262,6 +300,16 @@ export default defineComponent({
       );
       this.updateBasic();
     },
+    selectProgram(acronym: string) {
+      this.programAcronym = acronym;
+      this.updateBasic();
+    },
+    // remove college and all current majors
+    removeCollege() {
+      this.collegeAcronym = '';
+      this.majorAcronyms = [''];
+      this.updateBasic();
```

Comment:
I would also include `this.minorAcronyms = [''];` because we want to clear out the minors as well when we delete college.

https://user-images.githubusercontent.com/54298311/118316701-252a0580-b4c5-11eb-8a2f-1104f6d97963.mov

Reply:
Discussed over Slack. Going to make this change so that if a user removes their college, they remove their minors, as we do not want pure grad students to have minors. Will also disallow the dropdown from being opened if a college is not selected, similarly to how major dropdowns work now.